proposals/nnnn-configurable-time-before-shutdown.md

```diff
@@ -0,0 +1,78 @@
+# Configurable time before shutdown
+
+* Proposal: [SDL-NNNN](nnnn-configurable-time-before-shutdown.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+To prevent missing logs in SDL shutdown add Additional ini file options:
```

Comment:
@LuxoftAKutsan better to rephrase:
"To prevent missing logs after SDL shutdown, additional ini file options should be added:"